scripting/SurfTimer-discord.sp

```diff
@@ -338,12 +338,9 @@ public void SendBugReport(int iClient, char[] szText)
 	char szPlayerID[256], szSteamId[64], szName[MAX_NAME_LENGTH];
 	GetClientName(iClient, szName, sizeof szName);
 
-	if (g_cvProfileUrlType.BoolValue)
```

Comment:
Deleting this `if` statement removes the use of `g_cvProfileUrlType`. Some people might be relying on that for their panel (I do).

What you can do instead is move the `Format` call in the `if` statement. This will preserve the feature.

Reply:
`g_cvProfileUrlType` is used for steam/ own webstats so I will have to create a new bool cvar with steam2 / steam64 to make it work with panels using steam2 id

Reply:
Is it better to use steam2 as default value to not break existing sites or use steam64 to make it compatible with Surftimer Webstats ?